wfe/web-front-end_test.go

```diff
@@ -1231,6 +1263,50 @@ func TestGetCertificate(t *testing.T) {
 	test.AssertEquals(t, responseWriter.Header().Get("Content-Type"), "application/pkix-cert")
 	test.Assert(t, bytes.Compare(responseWriter.Body.Bytes(), certBlock.Bytes) == 0, "Certificates don't match")
 
+	// Valid request ID, cached
+	responseWriter = httptest.NewRecorder()
+	path, _ = url.Parse("/acme/cert/good-good-good-good-good-good-good-good-xxx")
+	wfe.Certificate(responseWriter, &http.Request{
+		Method: "GET",
+		URL:    path,
+	})
+	test.AssertEquals(t, responseWriter.Code, 200)
+	test.AssertEquals(t, responseWriter.Header().Get("Cache-Control"), "public, max-age=10")
+	test.Assert(t, bytes.Compare(responseWriter.Body.Bytes(), certBlock.Bytes) == 0, "Certificates don't match")
```

Comment:
Here's where we should check for the `Content-Location` header field, and verify that it returns the same thing.